### PR TITLE
Added APNS Token documentation

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-10140

### CODE REVIEW

File: website/docs/configuring_variants.md [hunk 1]
```diff
@@ -29,13 +29,32 @@ Click on `create` and your webpush variant is ready!
...
+ To be able to send _push notifications_ to iOS, an _iOS variant_ must be created. That variant can be an _iOS (Certificate) Variant_ or an _iOS (APNS Token) Variant, 
+ depending on the way we are going to authenticate to the APNs: 
+
+ * with an APNs Certificate
+ * with an APNs Token
+
+ The first one is the legacy one: with the method you will require to have one certificate for each application and, furthermore,
+ you need two distinct certificates for the production and the development environments.
+ 
+ The second one is the new token based authentication, that makes authentication much simpler:
```
Comment (Aindriunannerb) [inline website/docs/configuring_variants.md]: Perhaps change  "that makes authentication much simpler"  to   "which makes authentication much simpler"

File: website/docs/configuring_variants.md [hunk 2]
```diff
@@ -29,13 +29,32 @@ Click on `create` and your webpush variant is ready!
...
 
-#### Apple App ID and SSL Certificate for APNs
- Before you are able to use Push Notifications on your iOS Application, you must configure the Apple Push Notification service (APNs) with a certificate. 
+ To be able to send _push notifications_ to iOS, an _iOS variant_ must be created. That variant can be an _iOS (Certificate) Variant_ or an _iOS (APNS Token) Variant, 
+ depending on the way we are going to authenticate to the APNs: 
+
+ * with an APNs Certificate
+ * with an APNs Token
+
+ The first one is the legacy one: with the method you will require to have one certificate for each application and, furthermore,
```
Comment (Aindriunannerb) [inline website/docs/configuring_variants.md]: maybe change "with the method you will require to have " to  "with this method you will be required to have"

File: website/docs/configuring_variants.md (hunk 1 above)
Comment (ziccardi) [reply website/docs/configuring_variants.md]: 👍 Done!

File: website/docs/configuring_variants.md (hunk 2 above)
Comment (ziccardi) [reply website/docs/configuring_variants.md]: 👍 Done!

**Pull request merged**